plugins/auth-backend/src/service/router.ts

```diff
@@ -67,6 +71,28 @@ export async function createRouter(
 
   const appUrl = config.getString('app.baseUrl');
   const authUrl = await discovery.getExternalBaseUrl('auth');
+  const cacheMemoryType = config.getString('backend.cache.store');
```

Comment:
I believe this is optional to configure currently and should not be required here.

Reply:
this is how we can know what type of memory does client wants